Add instructions to setup APPUiO Keycloak extensions (#34)
appuio/appuio-io-docs: docs/modules/ROOT/pages/how-to/keycloak-brokering.adoc

Valid Redict URIs = https://{appuio-keycloak}/auth/realms/{appuio-realm}/broker/{foreign-idp-alias}/endpoint
----

. Add the group memberships to the tokens.
  Go to the "Mappers" tab and create a new Mapper.
+
[source,subs="attributes+"]
----
Name = group membership
Mapper type = Group Membership
Token Claim Name = groups
Full group path = false <1>
----

> <1> If this would be `true` (default), then a group name like `/Company/admins` would contain group `/Company/admins`, which might be undesirable.

Reviewer: This sentence is odd, I think you wanted to say something like:

```suggestion
<1> If this is `true` (the default), then the group name for group `/Company/admins` is `/Company/admins`, which might be undesirable.
```

>     If `false`, the group name would be just the group name of the leaf node, for example `admins`.

Reviewer: ```suggestion
    If this is `false`, the group name is just the value of the leaf node instead, for example group `/Company/admins` will be assigned the name `admins`.
```
